README.md

```diff
@@ -1,2 +1,5 @@
+[![Build Status](https://travis-ci.org/rjbautis/data-object-service-plugin.svg?branch=master)](https://travis-ci.org/rjbautis/data-object-service-plugin)
+
+
 # data-object-service-plugin
 Dockstore Data Object Service file provisioning plugin
```

Comment:
Optionally, it would be nice to describe this plugin a little more. Have a link to DOS repo, show a sample input like the s3 plugin. Sorry I missed this the first time I reviewed.